src/Data/Food/Recipe.elm

```diff
@@ -668,6 +669,23 @@ resetDistribution query =
     { query | distribution = Nothing }
 
 
+toStepsImpacts : Definition.Trigram -> Results -> Impact.StepsImpacts
+toStepsImpacts trigram results =
+    let
+        getImpact =
+            Impact.getImpact trigram
+                >> Just
+    in
+    { materials = getImpact results.recipe.ingredientsTotal
+    , transform = getImpact results.recipe.transform
+    , packaging = getImpact results.packaging
+    , transports = getImpact results.transports.impacts
+    , distribution = getImpact results.distribution.total
+    , usage = getImpact results.preparation
+    , endOfLife = Nothing
```

Comment:
I believe you added the `endOfLife` here for future comparison with the textile?

Reply:
Exactly, note that `StepsImpacts` is just a renaming of your previous `Steps` in the view, I've just made it a generic and transverse type of impacts.